X_train:  age                  0
workclass         1836
education_num        0
marital_status       0
occupation        1843
relationship         0
race                 0
sex                  0
capital_change       0
hours_per_week       0
native_country       0
dtype: int64
Index(['workclass', 'occupation'], dtype='object')
Traceback (most recent call last):
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular/train.py", line 99, in <module>
    'model_graph': wandb.Image(graph.render())
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/wandb/sdk/data_types/image.py", line 156, in __init__
    self._initialize_from_path(data_or_path)
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/wandb/sdk/data_types/image.py", line 252, in _initialize_from_path
    self._image = pil_image.open(path)
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/PIL/Image.py", line 3031, in open
    "cannot identify image file %r" % (filename if filename else fp)
PIL.UnidentifiedImageError: cannot identify image file 'Digraph.gv.pdf'